src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/RequestResponseMultipleEventArgs.cs

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.Security
+{
+    /// <summary>
+    /// This class is an event argument of the RequestResponse event.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+
+    public class RequestMultipleResponseEventArgs : EventArgs
+    {
+        /// <summary>
+        /// The cause of a triggered response.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public CallCause Cause { get; internal set; }
+
+        /// <summary>
+        /// The results of a permission request.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public IEnumerable<RequestResult> Results { get; internal set; }
+
+        /// <summary>
+        /// The privileges for which a permission was requested for.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public IEnumerable<string> Privileges { get; internal set; }
+    }
```

Comment:
It is uncomfortable to use. 
Let's see below use case
``` c#
void Handler(RequestMultipleResponseEventArgs arg) {
   var privileges = arg.Privileges.ToList();
   int index = 0;
   foreach (var result in arg.Results) {
       Console.WriteLine($"Result {privileges[index]} : {result}");
       index++;
   }
}
```
Results and Privileges should be a pair becuase it is a related data. 

Consider change to IList from IEnumerable
or Make a some class to combine `Result` and `Privilege`
``` c#
class Response {
    public RequestResult result { get; }
    public string privilege { get; }
}

public class RequestMultipleResponseEventArgs : EventArgs
{
     public CallCause cause { get; }
     public IEnumerable<Response> Responses {get; }
}
public class RequestResponseEventArgs : EventArgs
{
     public CallCause cause { get; }
     public Response Response {get; }
}

```

Reply:
I can change is for `RequestMultipleResponseEventArgs` class.
`RequestResponseEventArgs` was introduced in tizen 4. We don`t want to break backwards compatibility so I could extend it.
```csharp
public class RequestResponseEventArgs : EventArgs
    {
        public CallCause cause { get; internal set; }

        public RequestResult result { get; internal set; }

        public string privilege { get; internal set; }
        public Response Response {get; }
    }
```